Packs/AWS-IAMIdentityCenter/Integrations/AWSIAMIdentityCenter/AWSIAMIdentityCenter.py

```diff
@@ -0,0 +1,301 @@
+                                                                                  'groupName')))
+
+
+def remove_user_from_groups(args, client):  # pragma: no cover
+    membershipID = list_groups_for_user(args, client)
+    response = client.delete_group_membership(
+        IdentityStoreId=f'{IDENTITYSTOREID}',
+        MembershipId=f'{membershipID}'
+    )
+    if response['ResponseMetadata']['HTTPStatusCode'] == 200:
+        demisto.results(
+            "The User {0} has been removed from the group {1}".format(args.get('userName'),
+                                                                      args.get('groupName')))
+
+
+def test_function(client):
+    response = client.list_users()
+    if response['ResponseMetadata']['HTTPStatusCode'] == 200:
+        demisto.results('ok')
+
+
+def main():     # pragma: no cover
+    params = demisto.params()
+    aws_default_region = params.get('defaultRegion')
+    aws_role_arn = params.get('roleArn')
+    aws_role_session_name = params.get('roleSessionName')
+    aws_role_session_duration = params.get('sessionDuration')
+    aws_role_policy = None
+    aws_access_key_id = params.get('credentials', {}).get('identifier') or params.get('access_key')
+    aws_secret_access_key = params.get('credentials', {}).get('password') or params.get('secret_key')
+    verify_certificate = not params.get('insecure', True)
+    timeout = params.get('timeout')
+    retries = params.get('retries') or 5
+
+    validate_params(aws_default_region, aws_role_arn, aws_role_session_name, aws_access_key_id,
+                    aws_secret_access_key)
+
+    aws_client = AWSClient(aws_default_region, aws_role_arn, aws_role_session_name, aws_role_session_duration,
+                           aws_role_policy, aws_access_key_id, aws_secret_access_key, verify_certificate, timeout,
+                           retries)
+    command = demisto.command()
+    args = demisto.args()
+    client = aws_client.aws_session(
+        service=SERVICE,
+        role_arn=args.get('roleArn'),
+        role_session_name=args.get('roleSessionName'),
+        role_session_duration=args.get('roleSessionDuration'),
+    )
+
+    try:
+        demisto.debug('Command being called is {command}'.format(command=command))
+        if command == 'test-module':
+            test_function(client)
+        elif command == 'aws-iam-identitycenter-create-user':
+            create_user(args, client)
+        elif command == 'aws-iam-identitycenter-get-user':
+            get_user(args, client)
+        elif command == 'aws-iam-identitycenter-get-user-by-email':
+            get_user_by_email(args, client)
+        elif command == 'aws-iam-identitycenter-list-users':
+            list_users(args, client)
+        elif command == 'aws-iam-identitycenter-list-groups':
+            list_groups(args, client)
+        elif command == 'aws-iam-identitycenter-get-group':
+            get_user(args, client)
```

Comment:
Is this the correct method for the command **aws-iam-identitycenter-get-group**?

Reply:
Good catch thank you, I fixed it